components/org.wso2.carbon.identity.oauth.uma.grant/pom.xml

```diff
@@ -79,6 +79,7 @@
 
                             org.wso2.carbon.identity.core.util.*;version="${carbon.identity.framework.imp.pkg.version.range}",
                             org.wso2.carbon.identity.application.authentication.framework.model.*,
+                            org.wso2.carbon.identity.application.authentication.framework.util.*,
```

Comment:
Add specific imports instead of add all.

Reply:
This is the final package of the relevant class